app/sentry.ts

```diff
@@ -1,17 +1,24 @@
 import * as Sentry from '@sentry/browser';
-import { Ember } from '@sentry/integrations';
+import { CaptureConsole, Dedupe, Ember } from '@sentry/integrations';
 import config from 'open-event-frontend/config/environment';
 
 Sentry.init({
-    integrations: [new Ember()],
+    integrations: [
+        new Ember(),
+        new Dedupe(),
+        new CaptureConsole({
+            levels: ['error']
+        })
+    ],
     beforeSend(event: Sentry.Event) {
         const exception = event.exception?.values?.[0];
         const errorValue = exception?.value;
-        if (errorValue?.includes("Ember Data Request") && 
-            errorValue?.includes("404")) {
-            // Ignore 404 errors from Ember Data because 
-            // I don't know how to turn them off
-            return null;
+        if (errorValue?.includes("Ember Data Request")) {
+            if (errorValue?.includes("404")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 spaces but found 12.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5114150)